src/main/services/steam.service.ts

```diff
@@ -42,20 +44,28 @@ export class SteamService{
 
   public async getSteamPath(): Promise<string>{
 
-    if(!!this.steamPath){ return this.steamPath; }
-
-    const [win32Res, win64Res] = await Promise.all([
-      regedit.promisified.list(['HKLM\\SOFTWARE\\Valve\\Steam']),
-      regedit.promisified.list(['HKLM\\SOFTWARE\\WOW6432Node\\Valve\\Steam'])
-    ]);
-
-    const [win32, win64] = [win32Res["HKLM\\SOFTWARE\\Valve\\Steam"], win64Res["HKLM\\SOFTWARE\\WOW6432Node\\Valve\\Steam"]];
-    
-    let res = '';
-    if(win64.exists && win64?.values?.InstallPath?.value){ res = win64.values.InstallPath.value as string; }
-    else if(win32.exists && win32?.values?.InstallPath?.value){ res = win32.values.InstallPath.value as string; }
-    this.steamPath = res;
-    return this.steamPath;
+    if(this.steamPath){ return this.steamPath; }
+
+    switch (process.platform) {
+      case "linux":
+        this.steamPath = path.join(os.homedir(), '.steam', "steam");
```

Comment:
you can use electron instead of "os" to get homedir 
https://stackoverflow.com/a/54387144